runtime/src/consensus/tendermint/verifier.rs

```diff
@@ -640,20 +642,20 @@ impl VotingPowerCalculator for DomSepVotingPowerCalculator {
             let sign_bytes = signed_vote.sign_bytes();
             // Use Oasis Core domain separation scheme.
             let sign_bytes = Hash::digest_bytes_list(&[TENDERMINT_CONTEXT, &sign_bytes]);
-            if validator
+            let power = validator.power();
```

Comment:
The below: `if validator._.is_err(){ return Err()}` changed to `?`, but since the `validator` is moved into `map_err`  it complains if it is used in line 658

Reply:
oh moved :thinking: 

whereas before, it was smart enough to know that if we didn't return, we wouldn't have moved `validator`

tragic, but thanks for the explanation